flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/RemoteInputChannel.java

```diff
@@ -99,7 +100,7 @@
 	/** The number of available buffers that have not been announced to the producer yet. */
 	private final AtomicInteger unannouncedCredit = new AtomicInteger(0);
 
-	/** The number of required buffers that equals to sender's backlog plus initial credit. */
+	/** The number of buffers to requested that equals to unfulfilled sender's backlog. */
 	@GuardedBy("bufferQueue")
 	private int numRequiredBuffers;
```

Comment:
nit: might rename to `numRequiredFloatingBuffers` for better reflecting the current semantic.